src/dashboard/Makefile

```diff
@@ -131,7 +131,8 @@ C_SRCS_TEST = $(wildcard $(patsubst %, %/*.$(C_EXT), . $(TEST_SRCS_DIRS))) src/c
 # Write Configuration
 #=============================================================================#
 
-COMPORT ?= $(word 1, $(wildcard /dev/tty.usbserial-*) $(wildcard /dev/ttyUSB*))
+#COMPORT ?= $(word 1, $(wildcard /dev/tty.usbserial-*) $(wildcard /dev/ttyUSB*))
+COMPORT ?= /dev/ttyS4
```

Comment:
pls no

Reply:
export COMPORT=whatever

Reply:
Yeah dont commit this. If you dont want to export every time then just make sure not to commit this change.